compiler-rt/lib/asan/asan_stack.h

```diff
@@ -60,8 +75,16 @@ u32 GetMallocContextSize();
 #define GET_STACK_TRACE_MALLOC                                                 \
   GET_STACK_TRACE(GetMallocContextSize(), common_flags()->fast_unwind_on_malloc)
 
+#define GET_STACK_TRACE_MALLOC_WIN(pc, bp, caller_pc, extra_context)      \
```

Comment:
If they are rely _WIN then maybe move them into compiler-rt/lib/asan/asan_stack_win.h
If not, maybe rename to something general.

Reply:
Changes in this file are unused, please revert